mathics/format/asy.py

```diff
@@ -19,6 +19,7 @@
 
 from mathics.builtin.box.graphics3d import (
     Graphics3DElements,
+    Arrow3DBox,
```

Comment:
Why import this if it is not used later on? 

Are you planning on implementing this for Asymptote?

Reply:
@rocky to the truth I forget about this, tomorrow I'll have time to add the Asympytose version.